### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by shasherazi**
In this PR, I converted the project into a cli tool. It asks for and displays information via the command line.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  when '5'
+    app.create_rental
+  when '6'
+    app.list_rentals_for_person_id
+  when '7', 'q', 'quit'
+    break
+  end
+end
+
+puts 'Thank you for using this app!'
```
Comment (gemmen29) [inline main.rb]: - [x] Kindly move this code for starting the app inside a main function and call it at the end of your file as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=this%20will%20be%20a%20method%20called%20main%20that%20is%20invoked%20at%20the%20end%20of%20your%20file.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @shasherazi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb [hunk 2]
```diff
@@ -2,28 +2,4 @@
```
Comment (tufoinnkuo10) [inline main.rb:6]: - [x] Great job so far. please, what the previous reviewer meant was that, in your `main.rb` file you should add a method `main` like mentioned in your project requirement. this method will contain a list of options for the user to chose and how to quit like you did and finally at the bottom of the file the main method is invoked. 
you are almost done. keep up the good work.
eg 
```ruby
def main
  app = App.new()
...
...
...

end
end


main()
```

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @shasherazi 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: main.rb (hunk 2 above)
Comment (shasherazi) [reply main.rb:6]: Cant believe i forgot this, thanks for the review

**Review by prolajumokeoni — APPROVED** [review]
Hi @shasherazi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented 👏🏿   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**